enhancements/cudn-localnet/README.md

```
@@ -0,0 +1,562 @@
+  type: cudn-localnet
+  aap_job_template: osac_cudn_localnet
+  description: "CUDN Localnet — bridges VMs to fabric via VLAN tagging"
+```
+
+### Implementation Details/Notes/Constraints
+
+#### Ansible Role Structure
+
+The role lives in `osac-aap` and follows the existing template pattern:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)